test/e2e-go/features/followerNode/syncRestart_test.go

```diff
@@ -0,0 +1,127 @@
+	a := require.New(fixtures.SynchronizedTest(t))
+
+	// Overview of this test:
+	// Start a two-node network--one in follower mode (follower has 0%, secondary has 100%)
+	// with the nodes having a max account lookback of 2.
+	// Advance the primary node to particular rounds, set the follower's sync round
+	// and then advance the follower node as much as possible.
+	// Restart the network and verify that the sync round hasn't advanced.
+
+	// NOTE: with a max account lookback of MAL, and the follower's sync round at SR:
+	// 		 the follower cannot advance past round SR - 1 + MAL
+
+	var fixture fixtures.RestClientFixture
+	fixture.Setup(t, filepath.Join("nettemplates", "TwoNodesFollower100SecondMaxAccountLookback2.json"))
+
+	defer fixture.Shutdown()
+
+	// sanity check that the follower has the expected max account lookback of 2:
+	followerCtrl, err := fixture.GetNodeController("Follower")
+	a.NoError(err)
+	cfg, err := config.LoadConfigFromDisk(followerCtrl.GetDataDir())
+	a.NoError(err)
+	a.Equal(uint64(2), cfg.MaxAcctLookback)
+
+	waitTill := func(node string, round uint64) {
+		controller, err := fixture.GetNodeController(node)
+		a.NoError(err)
+		err = fixture.ClientWaitForRoundWithTimeout(fixture.GetAlgodClientForController(controller), round)
+		a.NoError(err)
+	}
+
+	getAlgod := func(node string) client.RestClient {
+		controller, err := fixture.GetNodeController(node)
+		a.NoError(err)
+		algod := fixture.GetAlgodClientForController(controller)
+		return algod
+	}
+
+	getRound := func(node string) uint64 {
+		algod := getAlgod(node)
+		status, err := algod.Status()
+		a.NoError(err)
+		return status.LastRound
+	}
+
+	getSyncRound := func() uint64 {
+		followClient := getAlgod("Follower")
+		rResp, err := followClient.GetSyncRound()
+		a.NoError(err)
+		return rResp.Round
+	}
+
+	a.Equal(uint64(1), getSyncRound())
+
+	waitTill("Primary", 3)
+	// with a max account lookback of 2, and the sync round at 1,
+	// the follower cannot advance past round 2 = 1 - 1 + 2
+	waitTill("Follower", 2)
+	a.LessOrEqual(uint64(3), getRound("Primary"))
+	a.Equal(uint64(2), getRound("Follower"))
+	a.Equal(uint64(1), getSyncRound())
+
+	/** restart the network **/
+	fixture.ShutdownImpl(true)
+	fixture.Start()
+
+	a.LessOrEqual(uint64(3), getRound("Primary"))
+	a.Equal(uint64(1), getSyncRound())
+	a.Equal(uint64(2), getRound("Follower"))
+
+	waitTill("Primary", 6)
+	followerClient := getAlgod("Follower")
+	err = followerClient.SetSyncRound(uint64(3))
+	a.NoError(err)
+	a.Equal(uint64(3), getSyncRound())
+	// with a max account lookback of 2, and the sync round at 3,
+	// the follower cannot advance past round 4 = 3 - 1 + 2
+	waitTill("Follower", 4)
+	a.LessOrEqual(uint64(6), getRound("Primary"))
+	a.Equal(uint64(4), getRound("Follower"))
+	a.Equal(uint64(3), getSyncRound())
+
+	fixture.ShutdownImpl(true)
+	fixture.Start()
+
+	a.LessOrEqual(uint64(6), getRound("Primary"))
+	a.Equal(uint64(4), getRound("Follower"))
+	a.Equal(uint64(3), getSyncRound())
```

Comment:
```suggestion
```
Although I like this portion of the test and am proposing keeping it, with a little nudge I'm happy to delete and then reduce the burden on CI.

Reply:
Not sure what the conventions are here. I share your sentiment, I like this part of the test too.